src/games/strategy/triplea/ai/proAI/data/ProMyMoveOptions.java

```diff
@@ -56,4 +59,8 @@ public ProMyMoveOptions(final ProMyMoveOptions myMoveOptions) {
     return transportList;
   }
 
+  public Map<Unit, Set<Territory>> getBomberMoveMap() {
```

Comment:
I think there could be a number of improvements made here. Some suggestions:
- Use guava `BiMap<Unit, Territory>` instead of `Map<Unit, Set<Territory>>`
- Poor abstraction, direct access to data representation, which is also poor encapsulation. Simple DataBean classes are an anti-pattern. For example, this move map is accessed in only a couple of ways. 

For example, consider the code below - most that could be moved to this class. There would be greater cohesion, since things related to the bomber move map would be in one class. As is, the bomber move map logic can be scattered anywhere throughout the code. So, the below seems like it would fit in a method similar to: `boolean canAnyTerritoriesBeBombed(Unit)`. It looks like only the one parameter would be needed.

```
+      // Check to see if any territories can be bombed
 +      final Map<Unit, Set<Territory>> bomberMoveMap = territoryManager.getAttackOptions().getBomberMoveMap();
 +      for (final Unit unit : bomberMoveMap.keySet()) {
 +        if (alreadyAttackedWithUnits.contains(unit)) {
 +          continue;
 +        }
 +        Optional<Territory> maxBombingTerritory = Optional.empty();
 +        int maxBombingScore = 3;
 +        for (final Territory t : bomberMoveMap.get(unit)) {
 +          final boolean territoryCanBeBombed = t.getUnits().someMatch(Matches.UnitCanProduceUnitsAndCanBeDamaged);
 +          if (territoryCanBeBombed && canAirSafelyLandAfterAttack(unit, t)) {
 +            final int noAABombingDefense = t.getUnits().someMatch(Matches.UnitIsAAforBombingThisUnitOnly) ? 0 : 1;
 +            int maxDamage = 0;
 +            final TerritoryAttachment ta = TerritoryAttachment.get(t);
 +            if (ta != null) {
 +              maxDamage = ta.getProduction();
 +            }
 +            final int numExistingBombers = attackMap.get(t).getBombers().size();
 +            final int remainingDamagePotential = maxDamage - 3 * numExistingBombers;
 +            final int bombingScore = (1 + 9 * noAABombingDefense) * remainingDamagePotential;
 +            if (bombingScore > maxBombingScore) {
 +              maxBombingScore = bombingScore;
 +              maxBombingTerritory = Optional.of(t);
 +            }
 +          }
 +        }
```